src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -84,6 +85,17 @@ public function __construct(ResourceClassResolverInterface $resourceClassResolve
         $this->graphQlSubscriptionManager = $graphQlSubscriptionManager;
         $this->graphQlMercureSubscriptionIriGenerator = $graphQlMercureSubscriptionIriGenerator;
         $this->reset();
+
+        $urlencode = ExpressionFunction::fromPhp('urlencode');
```

Comment:
We should use `rawurlencode`, `utlencode` is intended to be used to create request bodies (forms). https://www.php.net/manual/en/function.rawurlencode.php

Reply:
Do you think I leave it as "rawurlencode" or do an alias for "url"?

Reply:
Yes